tests/snuba/api/endpoints/test_organization_events_stats.py

```diff
@@ -366,7 +365,7 @@ def test_throughput_eps_minute_rollup(self):
             )
         assert response.status_code == 200, response.content
         data = response.data["data"]
-        assert len(data) == 7
+        assert len(data) == 6
```

Comment:
@iProgramStuff This should be 7 data points.

Given:

```
            response = self.client.get(
                self.url,
                format="json",
                data={
                    "start": iso_format(self.day_ago),
                    "end": iso_format(self.day_ago + timedelta(minutes=6)),
                    "interval": "1m",
                    "yAxis": "eps()",
                    "project": project.id,
                },
            )
```

```
assume:
start = 12:00:00
end = 12:06:00 

data point 1 = 12:00:00 (start)
data point 2 = 12:01:00
data point 3 = 12:02:00
data point 4 = 12:03:00
data point 5 = 12:04:00
data point 6 = 12:05:00
data point 7 = 12:06:00 (end)
```

As you can see from above, there are 7 data points. `start` and `end` are inclusive ranges.

Reply:
Let me take a closer look at what's happening here and get back to you.

Reply:
It should be end exclusive https://github.com/getsentry/sentry/pull/15939